Xamarin.Forms.Core/Slider.cs

```diff
@@ -118,9 +135,34 @@ public double Value
 
 		public event EventHandler<ValueChangedEventArgs> ValueChanged;
 
+		public event EventHandler DragStarted;
+		public event EventHandler DragCompleted;
+
 		public IPlatformElementConfiguration<T, Slider> On<T>() where T : IConfigPlatform
 		{
 			return _platformConfigurationRegistry.Value.On<T>();
 		}
+
+		public void SendDragStarted()
```

Comment:
should be attributed with `[EditorBrowsable(EditorBrowsableState.Never)]`

Reply:
Done.